From 2dd60c1c32d4b40530e7fb25a089e5f8bf0aef5d Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:26 -0400
Subject: [alsa] hda: Add another headset pin quirk for some Dell machines

Message-id: <140972823573-367-patchset-jkysela@redhat.com>
Patchwork-id: 91063
O-Subject: [RHEL 7.1 PATCH 366/442] ALSA: hda - Add another headset pin quirk for some Dell machines
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Another quirk to make the headset mic work on some new Dell machines.

Cc: Hui Wang <hui.wang@canonical.com>
BugLink: https://bugs.launchpad.net/bugs/1297581
Signed-off-by: David Henningsson <david.henningsson@canonical.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: e03fdbde8aca2f6fe795803e4eccbc156d1eec5b)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 60bf5ee..4ebbe03 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -5199,6 +5199,18 @@ static const struct snd_hda_pin_quirk alc269_pin_fixup_tbl[] = {
   {0x1b, 0x411111f0},
   {0x1d, 0x40700001},
   {0x1e, 0x411111f0}),
+ SND_HDA_PIN_QUIRK(0x10ec0293, 0x1028, "Dell", ALC293_FIXUP_DELL1_MIC_NO_PRESENCE,
+  {0x12, 0x40000000},
+  {0x13, 0x90a60140},
+  {0x14, 0x90170110},
+  {0x15, 0x0221401f},
+  {0x16, 0x411111f0},
+  {0x18, 0x411111f0},
+  {0x19, 0x411111f0},
+  {0x1a, 0x411111f0},
+  {0x1b, 0x411111f0},
+  {0x1d, 0x40700001},
+  {0x1e, 0x411111f0}),
  {}
 };
 
-- 
1.7.1